src/js/app.js

```
@@ -2476,19 +2515,117 @@ document.addEventListener('DOMContentLoaded', () => {
-        if (orgsSec) orgsSec.scrollIntoView({ behavior: 'smooth' });
-        applyFilters();
+  const heroResults = document.getElementById('heroSearchResults');
+  if (heroSearch && heroResults) {
+    let heroActiveIdx = -1;
+
+    function closeHeroDropdown() {
+      heroResults.classList.remove('open');
+      heroSearch.setAttribute('aria-expanded', 'false');
+      heroActiveIdx = -1;
+      heroSearch.removeAttribute('aria-activedescendant');
+    }
```

ESLint

[error] Move function declaration to function body root.

(no-inner-declarations)